### ISSUE TITLE
Runepriest - "Mountain Spirit" & "Regenrative Shielding"

### ISSUE THREAD

**Issue by Marffff** [WarEmu/WarBugs#7956]
Morale "Mountain Spirit" doesn't stack with armor pot or tactic "Regenerative Shielding" or kotbs resi aura.

Tactic "Regenrative Shielding" doens't stack with resi pot or morale, is it tactic or is it buff/skill ?
No "armor penetration" in the buff tracker/disrcription.

Both don't seems to work right.

Azarael wrote:
[MAX (Ability / Potion) ] ................................armor pot/ kotbs resi aura
+                                                              +
[MAX (Morale)] ................................................."Mountain Spirit"
+                                                              +
[SUM (Tactic)] ..................................................."Regenrative Shielding" if it is a "tactic"
+ 
[SUM (Career Mechanic, Heaven's Blade, Daemonic Armor, Flak Jacket)]

**Comment by Pxl555** [comment]
Defensive buff morals that increase your 'groups' armor/resistances are an exception to the rule. As for Regenerative Shielding you have this #2537

**Comment by Luth-RoR** [comment]
Not sure, but i think there is a misunderstanding: as i read it MAX means that only the highest value is taken and SUM means that the values are additive. This means that 2x MAX doesn't stack (like on the mythic server).

**Comment by Pxl555** [comment]
it has plus inbetween :D

**Comment by Luth-RoR** [comment]
Yes that would mean, for instance, 600 armor pot + 1000 morale = + 1000 armor. I think we have to see that from a coder perspective.
